content/25.discussion.md

```diff
@@ -1,37 +1,45 @@
 ## Discussion
 
-A major challenge that we faced in carrying out this work was to narrow down geographic origins for some groups of names.
+There are significant technical and ethical challenges that one faces in carrying out retrospective work to examine the fairness of scientific practices.
+A major technical challenge was to narrowing down geographic origins for some groups of names.
 Some name origin groups, such as Hispanic names, are geographically disparate.
-We were unable to construct a classifier that could distinguish between names from Iberian countries (Spain and Portugal) from those in Latin America in the group of Hispanic names.
+We were unable to construct a classifier that could distinguish names from Iberian countries (Spain and Portugal) from those in Latin America in the group of Hispanic names.
 Discrepancies in representation between these groups are thus undetectable by our classifier.
 Honoree counts of those with Hispanic names are influenced from Spain as well as Latin America.
 In such cases, our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.
-
-Biases in authorship practices may also result in our underestimation of the composition of minoritized scientists within the field.
-We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
+Another technical challenge is that supervised machine learning approaches are neither error free nor bias free.
+By integrating different lines of evidence and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we aimed to alleviate method-specific biases and discover insightful findings that correctly reflect the current representation diversity at conferences.
+
+A key ethical challenge with retrospective work to examine disparities is that algorithmic approaches to infer characteristics are often required.
+This leads to significant limitations such as considering gender as a binary variable.
+While this situation limits retroactive examination, there is a substantial opportunity for scientific societies, grant-making organizations, publishers, and others to proactively collect self-identified demographic information.
+A key element is then implementing processes to use these plans to continuously evaluate practices.
```

Comment:
I'm trying to find a way to say this right and avoid repetitions. Here's a suggestion but can be adjusted.

```suggestion
It is equally crucial that this information is properly used to continuously evaluate inclusion practices.
```